cms/djangoapps/contentstore/rest_api/v1/views.py

```diff
@@ -100,7 +99,6 @@ def get(self, request, course_id):
             data['proctored_exam_settings'] = proctored_exam_settings
             data['available_proctoring_providers'] = get_available_providers()
             data['course_start_date'] = course_metadata['start'].get('value')
-            data['is_staff'] = request.user.is_staff
```

Comment:
I don't fully understand how this fits into what we're doing in this diff, but I don't think that's a blocking concern.

Reply:
The team had talked about removing this piece of information, as we don't need it in the MFE. The user context provided to us by `frontend-platform` lets us know if the user is staff. We didn't have it ticketed - it was a "sure, let's do that at some point" thing, so I slid it into this pull request. I should have made a note of that in the pull request, though.